official-build.props

```diff
@@ -1,3 +1,3 @@
 DISABLE_AUTOTAGGING=true
 USE_OPEN_SOURCE_SETTINGS=false
-SEPG_BUILD_ENV_IMAGE=cafapi/opensuse-jdk8-maven:1.0.0
+SEPG_BUILD_ENV_IMAGE=cafapi/buildenv-jdk11:1.0.0
```

Comment:
If we're trying to move this project to Java 11 then we should also be updating the parent.  It is currently on [`2.4.0-254`](https://github.com/CAFapi/caf-common-parent/releases/tag/v2.4.0), but for Java 11 is should be moving to [`3.0.0-318`](https://github.com/CAFapi/caf-common-parent/releases/tag/v3.0.0).

https://github.com/CAFapi/caf-common-parent/compare/v2.4.0..v3.0.0